.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: tweag/linear-types:0.1.6
+    environment:
+      - stack: stack --no-docker
```

Comment:
I don't find the factoring out of `stack --no-docker` helpful. Just write `stack --no-docker` inline everywhere. Overzealous application of DRY sometimes has readability cost through obfuscation while bringing no benefit.

Reply:
Really? Because I find it positively obnoxious to write `--no-docker` everywhere. It is not part of the logic of the test, but of the overall architecture of the CI, so the repetition is obfuscation, in my view.

Reply:
Just change it. It's what we do in virtually every other CircleCI deployment we have (usually concerning the `--nix` argument).